plugin/doc_comment.js

```diff
@@ -390,13 +390,15 @@
 
     for (var i = 0; i < comments.length; ++i) {
       var comment = comments[i];
-      var decl = /(?:\n|\*)\s*@(type|param|arg(?:ument)?|returns?|this|class|constructor)\s+(.*)/g, m;
+      var decl = /(?:\n|\*)\s*@(type|param|arg(?:ument)?|returns?|this|class|constructor)(?:\s*?\n|\s+(.*))/g, m;
       while (m = decl.exec(comment)) {
         if (m[1] == "class" || m[1] == "constructor") {
           self = foundOne = true;
           continue;
         }
 
+        if (m[2] === undefined) continue; // to avoid tags that require a type argument.
```

Comment:
Is there any sample for this case?

Reply:
this makes sure that a tag without an argument (`@constructor` for example) won't be examined as tags that require one (`@param` for example).

Reply:
`constructor` already handled at line 395.
So this is just a second guard for any other tags(maybe new tag in the future) without argument?

Reply:
It is needed because other tags expect `m[2]` to be a string and it might not be the case. For example, in the param case below it is sliced.
this line continues to the next iteration to avoid an exception.